libelement/include/element/interpreter.h

```diff
@@ -289,50 +308,50 @@ element_result element_interpreter_compile(
  * @return ELEMENT_ERROR_API_INVALID_INPUT inputs pointer is null
  * @return ELEMENT_ERROR_API_OUTPUT_IS_NULL outputs pointer is null
  */
-element_result element_interpreter_evaluate(
+element_result element_interpreter_evaluate_declaration(
```

Comment:
this evaluates an instruction tree, not a declaration

Reply:
👍